blocks/header/header.css

```diff
@@ -42,10 +49,364 @@ header nav a:any-link {
+  line-height: 19.2px;
+  display: flex;
+  flex-direction: column;
+  gap: 13px;
+  font-weight: normal !important;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div ul{
+ position: absolute;
+ padding-top: 30px !important;
+ padding-bottom: 15px !important;
+ top: 10px;
+ left: 10px;
+ border-radius: 20px;
+ display: none !important;
+ max-height: 200px;
+ overflow-y: scroll;
+  
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div ul::-webkit-scrollbar{
+  display: none;
+}
+
+header .nav-menu-overlay .nav-menu-column.right>.submenu>li {
+  text-transform: uppercase;
+  font-weight: 800;
+  width: 90%;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div ul li{
+  display: block;
+  padding: 6px 20px;
+  height: 22px;
+  width: 100%;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .nav-menu-overlay-back::after {
+  content: "";
+  width: 17px;
+  position: absolute;
+  left: 13px;
+  height: 3px;
+  border-radius: 10px;
+  top: 18.5px;
+  background-color: black;
+  border: 3px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .submenu-main-title {
+  position: absolute;
+  margin: 0 auto;
+  width: 90%;
+  display: flex;
+  height: 60px;
+  justify-content: center;
+  z-index: 1;
+  word-wrap: break-word;
+  color: white;
+  font-size: 20px;
+  font-weight: 700;
+  text-align: center;
+  top: 40px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right {
+  flex-direction: column;
+  display: none;
+  min-width: max-content;
+}
+
+header .nav-menu-overlay .nav-menu-column.right>.submenu>li > ul {
+  padding: 0;
+  padding-top:15px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right> .submenu> li > ul > li > ul  {
+  display: flex;
+  flex-direction: column;
+  gap: 13px;
+  font-size: 16px;
+  padding-left: 20px;
+  margin-top: 10px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country input[type="text"] {
+  background-color: #004370;
+  outline: none;
+  border: none;
+  position: absolute;
+  top: 30px;
+  left: -170px;
+  width: 300px;
+  z-index: 3;
+  padding: 17px 10px 17px 35px;
+  border-radius: 40px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country input[type="text"]:focus + .country-list{
+  display: block !important;
+}
+
+.section.nav-sections .default-content-wrapper p {
+  font-size: var(--body-font-size-m);
+  color: var(--primary-blue-80);
+  text-decoration: underline;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country p{
+  position: absolute;
+  width: 30px;
+  height: 30px;
+  top: 65px;
+  background-color: #001C2D;
+  left: 300px;
+  z-index: 3;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country p::after{
+  content: '';
+  width: 2px;
+  display: inline-block;
+  padding: 2px;
+  height: 2px;
+  border-left: 2px solid #169af3;
+  border-bottom: 2px solid #169af3;
```

Comment:
Please see if if you use color variables.

Reply:
Done